packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -277,25 +277,25 @@ const EVENTS = {
     event: 'view',
   },
 
-  'screen.cannot_create_account': {
+  'screen.cannot-create-account': {
     group: GROUPS.registration,
     event: 'cannot_create_account_view',
   },
 
   // cookies_disabled
-  'screen.cookies_disabled': {
+  'screen.cookies-disabled': {
     group: GROUPS.activity,
     event: 'cookies_disabled_view',
   },
-  'cookies_disabled.submit': {
+  'flow.cookies-disabled.submit': {
     group: GROUPS.activity,
     event: 'cookies_disabled_submit',
   },
-  'cookies_disabled.success': {
+  'flow.cookies-disabled.success': {
     group: GROUPS.activity,
     event: 'cookies_disabled_success',
   },
-  'cookies_disabled.fail': {
+  'flow.cookies-disabled.fail': {
```

Comment:
Not relevant and possibly not interesting, but I had a few minutes to poke at these amplitude event names and wasn't sure if this was concerning (since we're on the topic of consistency):

```js
Object.entries(EVENTS)
  .forEach(([keyName, obj ]) => {
    const keyNameRegExp = /^[a-z\-.]*?$/;
    if (!keyNameRegExp.test(keyName)) {
      console.log("key:", keyName);
    }

    const eventRegExp = /^[a-z_]*?$/;
    if (!eventRegExp.test(obj.event)) {
      console.log("event:", obj.event);
    }
  });
```

### OUTPUT:
```sh
key: flow.choose-what-to-sync.cwts_do_not_sync
key: flow.would-you-like-to-sync.cwts_do_not_sync
key: enter-email.thirdPartyAuth
```

Where the first two are using underscores, and the last one is camelCase-instead-of-hyphens.

Reply:
Ah I should have read this before responding to you [here](https://github.com/mozilla/fxa/pull/14830#issuecomment-1414502160) lol.

Creating a check wouldn't be too bad if there's only 3 events we'd need to change but eh. I also kind of feel like it's not worth it since these are a mess and need to be re-evaluated anyway. 🫤

Reply:
Yeah, I'm fine with doing nothing if we're thinking of refactoring (or just doing nothing in general). I just wanted a pair of 👀 on the results above and make sure nobody was panicked or anything else needed changes in this PR.